app/services/resolve_account_service.rb

```diff
@@ -26,11 +26,10 @@ def call(uri, options = {})
 
     @account ||= Account.find_remote(@username, @domain)
 
-    return @account if @account&.local? || !webfinger_update_due?
+    return @account if @account&.local? || @domain.nil? || !webfinger_update_due?
```

Comment:
Is there a case where `@account.local?` would be false and `@domain.nil?` would be true?

Reply:
There would be a case where @account is nil *and* @domain is nil. It looks like before this, doing a local mention of a non-existing account attempted to perform a webfinger resolution (which failed due to invalid URI structure).

Reply:
alright